fw-child/template/query/news-item.php

```diff
@@ -36,7 +39,11 @@
 
 		<div class="card-date mb-1">
 			<?php
-			echo get_post_time( 'F j, Y', false, $item['id'] ); ?>
+
+				// Echo post time in current language
+				echo date_i18n( __( 'F j, Y', 'cdc' ), $post_time );
```

Comment:
Since the lines have moved, the .pot and .po file would need to be regenerated.

I would take advantage of this PR to also update the `fw-child/languages/cdc/fr_CA.po` file so that the French translation of `'F j, Y'` is simply `'j F, Y'`, not `'le j F, Y'` (i.e. "24 avril 2024" instead of "_le_ 24 avril 2024").